app/components/contents/structural_component.rb

```diff
@@ -22,25 +25,5 @@ def viewable?
     def paginatable_array
       @paginatable_array ||= Kaminari.paginate_array(structural.contains).page(params[:page]).per(50)
     end
-
```

Comment:
These lines were moved into the new StructuralPresenter unchanged for reuse, with added test coverage in the presenter spec.